### DISCUSSION TITLE
Mock url not show after importing Postman collection

### DISCUSSION THREAD

**Discussion by trinhpham** [microcks/microcks#633] [Q&A] [answered]
Hello,

I have a simple Postman collection like below

<details>
  <summary>Click to expand!</summary>
  
  ```json
{
	"info": {
		"_postman_id": "f6c5d0d1-3ca3-4ae6-b6d4-c78ccf8c1697",
		"name": "TestCol",
		"description": "version=v1 - Test",
		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json",
		"_exporter_id": "12970379"
	},
	"item": [
		{
			"name": "TestGet",
			"request": {
				"method": "GET",
				"header": [
					{
						"key": "header1",
						"value": "123",
						"type": "text"
					}
				],
				"url": "https://httpbin.org/get"
			},
			"response": []
		}
	]
}
  ```
</details>

I created an importer to import that file into Microcks, the import process went well but when viewing the API detail, I don't see its mock url as document (see image)
![image](https://user-images.githubusercontent.com/9128061/179957997-0623cd64-7d02-4520-b58f-4d32d4d5bd5e.png)

Am I missing something?

**Comment by trinhpham** [comment]
I found that I have to add example to the Postman request, so the parser can parse the file.
IMO, this should be a bug to fix :(

**Comment by lbroudoux** [comment, marked as answer]
Hi @trinhpham,

Mocks in Microcks are only based on provided examples alongside the contract (whether it is Postman Collection or Swagger/OpenAPI artifact). Mocks URLs are generated per example (even if they all share common prefix being `http://<microcks-url>/rest/<api-name>/<api-version`. So in a nutshell: no example means no available mock and no exposed mock url. 😉
